app/routes/users+/$username.tsx

```diff
@@ -64,8 +65,12 @@ export default function UsernameRoute() {
 					</p>
 					{isLoggedInUser ? (
 						<Form action="/logout" method="POST" className="mt-3">
-							<Button type="submit" variant="secondary" size="pill">
-								Logout
+							<Button type="submit" variant="link" size="pill">
+								<Icon
+									name="exit"
+									className="scale-125 max-md:scale-150 md:mr-2"
+								/>
+								<span>Logout</span>
```

Comment:
If the Icon is intended to be rendered next to text, you should use the children feature:

```suggestion
								<Icon
									name="exit"
									className="scale-125 max-md:scale-150"
								>
									Logout
								</Icon>
```

Reply:
Added mr-1 since, there is not much gap between icon and text.

Reply:
Hmmm... I prefer to keep things consistent. Could you show me a screenshot? And compare it with the delete button on a note?

Reply:
mr-2 is what we are using currently for note-edit and note-delete.

![image](https://github.com/epicweb-dev/epic-stack/assets/143390/fff27f52-a1a7-45ef-aec6-f2365511f8fb)

Also we do not have the icon text as child. 
```jsx
<Link to="edit">
  <Icon
	  name="pencil-1"
	  className="scale-125 max-md:scale-150 md:mr-2"
  />
  <span className="max-md:hidden">Edit</span>
</Link>
```

Should we change all of them to the one below?
```jsx
<Link to="edit">
  <Icon
	  name="pencil-1"
	  className="scale-125 max-md:scale-150 md:mr-2"
  >
    <span className="max-md:hidden">Edit</span>
  </Icon>
</Link>
```

Reply:
I didn't realize there was a margin on the delete button. Perhaps we should update the default `gap`s on Icon 🤔 https://github.com/epicweb-dev/epic-stack/blob/84537caaec3abbbd61658aecd4ac98319eb73686/other/svg-icon-template.txt#L20-L27